bundles/org.smarthomej.binding.deconz/src/main/resources/OH-INF/thing/sensor-thing-types.xml

```diff
@@ -555,12 +555,20 @@
 		<description>Status of this thermostat's childlock.</description>
 		<category>Lock</category>
 	</channel-type>
+	<channel-type id="windowopen">
+		<item-type>Contact</item-type>
+		<label>Window Open</label>
+	</channel-type>
+	<channel-type id="externalwindowopen">
+		<item-type>Switch</item-type>
```

Comment:
@cweitkamp I'm unhappy with this modeling as a "Window Open" usually is a `Contact`. But `OpenClosedType` is not a command and this is an input to an external device, so we need a command. Or do you think this is a corner-case where the thing handler should implement `handleUpdate`?

Reply:
Isn't `String` an option? With Command/StateOptions? This way you can even say that the external window can have more than two states like open, close and tilt.

Reply:
For the API it's mapped to a boolean, so there are only two possible states. We already have a `windowopen` status (for thermostats which have a direct-connected window contact) channel which uses `Contact` as item-type (like `open` channel for Open/Closed sensors). All other boolean states are `Switch` channels. I fear that having a `String` channel here is even more confusing than the `Switch`.

Reply:
The more I think about it, the more I think that `handleUpdate` would be the correct solution. What is essentially expected here is to pass a state update from a window contact to the thermostat. It's not a command but really a state.

Reply:
Forget my comment. I just noticed we don't have `handleUpdate` any longer.